composer.json

```diff
@@ -23,14 +23,14 @@
         "php": "^7.0.8",
         "symfony/framework-bundle": "^3.3|^4.0",
         "symfony/property-access": "^3.0|^4.0",
-        "symfony/translation": "^3.2|^4.0"
+        "symfony/translation": "^3.2|^4.0",
+        "alcaeus/mongo-php-adapter": "^1.1"
```

Comment:
This should be in `suggests` so that people only using the ORM/ODM/Elastic adapters don't have to install it. All adapter specific extensions should use runtime feature detection.

Further down the road, if we get enough adapters, we should likely put them in separate repositories, but for now it's good enough as a monolith.

Reply:
@curry684 I've done some test to try to resolve build errors.
I'll restore my previous commit.

Reply:
Okies, thanks for all the effort  👍